Sources/Turf/GeoJSON.swift

```diff
@@ -0,0 +1,358 @@
+import Foundation
+#if !os(Linux)
+import CoreLocation
+#endif
+
+
+public class BaseFeature: Codable {
+    private enum CodingKeys: String, CodingKey {
+        case properties
+    }
+    
+    public var properties: [String : AnyJSONType]?
+    
+    public required init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+        properties = try container.decode([String : AnyJSONType]?.self, forKey: .properties)
+    }
+    
+    public func encode(to encoder: Encoder) throws {
+        var container = encoder.container(keyedBy: CodingKeys.self)
+        try container.encode(properties, forKey: .properties)
+    }
+}
+
+public class Feature: BaseFeature {
```

Comment:
It seems problematic that features and feature collections would have class (by reference) semantics while geometries would have struct (by value) semantics.